lua/astrocommunity/file-explorer/mini-files/init.lua

```diff
@@ -1,3 +1,26 @@
+-- This function is taken from LazyVim: https://github.com/LazyVim/LazyVim/blob/cb223553ff73eb2f37ffb5dc0bb75b76a4677faf/lua/lazyvim/plugins/extras/editor/mini-files.lua
+local function map_split(buf_id, lhs, direction, close_on_file)
+  local files = require "mini.files"
+  local should_close = close_on_file == nil and true or close_on_file
+
+  local rhs = function()
+    -- Make new window and set it as target
+    local cur_target = files.get_explorer_state().target_window
+    local new_target = vim.api.nvim_win_call(cur_target, function()
+      vim.cmd(direction .. " split")
+      return vim.api.nvim_get_current_win()
+    end)
+
+    files.set_target_window(new_target)
+    files.go_in { close_on_file = should_close }
+  end
+
+  -- Adding `desc` will result into `show_help` entries
+  local desc = "Split " .. direction
```

Comment:
Copied from original solution. 
```suggestion
  local desc = "Open in " .. direction .. " split"
```